packages/catalog-generator/src/test/java/io/kaoto/camelcatalog/commands/GenerateCommandTest.java

```diff
@@ -84,7 +86,9 @@ void testGeneratorCalledWithCorrectParameters() {
             File expectedFolder = new File(tempDir, "camel-main/4.8.0");
             verify(builder, times(1)).withOutputDirectory(expectedFolder);
 
-            assertEquals(catalogDefinition.getFileName(), "camel-main/4.8.0/index.json");
+            String expectedFile = Stream.of("camel-main", "4.8.0", "index.json").reduce((a, b) -> a + File.separator + b).get();
```

Comment:
Is there a better way to do this? A more Java way

Reply:
i think it is more common to write it this way (at least for old timers):
```
"camel"+File.separator+"4.8.0"+File.separator+"index.json"
```

With more recent features of Java, I would look to something like: Paths.get("camel", "4.8.0", "index.json").toString()

Reply:
it seems `Path.get()` doesn't exist but `Path.of()` worked 👍

Reply:
It is `Paths` with plural

Reply:
Path.of() can do the trick too

Reply:
ah, plural 🤦‍♂️